frontend/app/src/assets/images/airdrops/FOX_Icon_white.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You added two svgs but only one is used. Why?

Reply:
ShapeShift provides two icons, one for light theme and another for dark. @kelsos might find them useful to customize the behaviour and improve the UI

Reply:
Okay but if he does not we shuld remove the extra svg. No need to make the final binary bigger without actually using stuff.